manpages/mkfs.exfat.8

```diff
@@ -67,12 +67,23 @@ _
 >128 GiB	\[<=]512 GiB	256 KiB	32 MiB
 >512 GiB	\[<=]2 TiB	512 KiB	64 MiB
 .TE
+The default is always 1 MiB.
```

Comment:
Might be a good idea to have some rationale here. Just the obvious "not every block device is an SD card" could do.